doc/10-icinga-template-library.md

```diff
@@ -2798,6 +2798,42 @@ Name                            | Description
 smart_attributes_config_path    | **Required.** Path to the smart attributes config file (e.g. check_smartdb.json).
 smart_attributes_device         | **Required.** Device name (e.g. /dev/sda) to monitor.
 
+### Hardware <a id="plugin-contrib-hardware-switch"></a>
+
+This category includes all plugin check commands for various network hardware checks.
```

Comment:
There's already a hardware section, isn't it?

Reply:
You are right!